src/compas_xr/mqtt/messages.py

```diff
@@ -0,0 +1,460 @@
+        instance = cls(increment_response_ID=False, sequence_id=sequence_id, response_id=response_id, device_id=device_id, time_stamp=time_stamp)
+
+        #Update the sequence counter and response id from the message received.
+        cls._update_sequence_counter_from_message(instance, sequence_id)
+        cls._update_response_id_from_message(instance, response_id)
+        
+        return instance
+    
+    def _get_device_id(self):
+        if not Header._device_id:
+            Header._device_id = str(uuid.uuid4())
+            self.device_id = Header._device_id
+        else:
+            self.device_id = Header._device_id
+        return self.device_id
+
+    def _get_time_stamp(self):
+        self.time_stamp = datetime.now().strftime("%Y-%m-%d %H:%M:%S.%f")
+        return self.time_stamp
+    
+    def _ensure_sequence_id(self):
+        if not Header._shared_sequence_counter:
+            Header._shared_sequence_counter = SequenceCounter()
+            self.sequence_id = Header._shared_sequence_counter._value
+        else:
+            self.sequence_id = Header._shared_sequence_counter.increment()
+        return self.sequence_id
+    
+    def _ensure_response_id(self, increment_response_ID=False):
+        if not Header._shared_response_id_counter:
+            Header._shared_response_id_counter = ResponseID()
+            self.response_id = Header._shared_response_id_counter._value
+        else:
+            if increment_response_ID:
+                self.response_id = Header._shared_response_id_counter.increment()
+                self.response_id = Header._shared_response_id_counter._value
+            else:
+                self.response_id = Header._shared_response_id_counter._value
+        return self.response_id
+    
+    def _update_sequence_counter_from_message(self, sequence_id):
+        if Header._shared_sequence_counter is not None:
+            Header._shared_sequence_counter.update_from_msg(sequence_id)
+        else:
+            Header._shared_sequence_counter = SequenceCounter(start=sequence_id)
+    
+    def _update_response_id_from_message(self, response_id):
+        if Header._shared_response_id_counter is not None:
+            Header._shared_response_id_counter.update_from_msg(response_id)
+        else:
+            Header._shared_response_id_counter = ResponseID(start=response_id)
+
+    @property
+    def data(self):
+        return {
+            "sequence_id": self.sequence_id,
+            "response_id": self.response_id,
+            "device_id": self.device_id,
+            "time_stamp": self.time_stamp,
+        }
+    
+class GetTrajectoryRequest(UserDict):
+    """Message objects used for publishing and subscribing to/from topics.
+
+    A message is fundamentally a dictionary and behaves as one."""
+
+    def __init__(self, element_id, robot_name, header=None):
+        # super(GetTrajectoryRequest, self).__init__()
+        if header is not None:
+            self.header=header
```

Comment:
Again, it seems black is not running on this file, because this should be:
```suggestion
            self.header = header
```